test/kind-config.yml

```diff
@@ -0,0 +1,7 @@
+  
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
```

Comment:
Please type an enter at the end of the file.

Reply:
Done